.gitignore

```diff
@@ -36,5 +36,4 @@ build/
 
 # Rust
 target
-Cargo.lock
```

Comment:
Why do you want to commit this file?

Reply:
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries